R-packages/modeltools/R/quantgen.R

```diff
@@ -106,7 +106,7 @@
 #'   `value`. The `quantile` column gives the probabilities associated with
 #'   quantile forecasts for that location and ahead. 
 #' 
-#' @importFrom dplyr filter select pull summarize between
+#' @importFrom dplyr filter select pull summarize between bind_cols
```

Comment:
I'm mildly surprised to find that `%>%` is not needed in this import, since it's not a built-in operator. Maybe `modeltools` is always used in an environment where `tidyr` is already loaded?

Reply:
Looks like another function must already `@importFrom %>%`, since it's in the `NAMESPACE`: https://github.com/cmu-delphi/covidcast/blob/6bca816b62e700cc90f31b0e3d5a6c0edda31da5/R-packages/modeltools/NAMESPACE#L15